VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/tests/participant-waiting-room.component.eventhub.events.spec.ts

```diff
@@ -53,7 +53,7 @@ describe('ParticipantWaitingRoomComponent event hub events', () => {
             userMediaStreamService,
             notificationSoundsService
         );
-        adalService.userInfo.userName = globalParticipant.username;
+        adalService.userInfo.userName = 'chris.green@hearings.net';
```

Comment:
same question (just wondering)

Reply:
we remove the username from ParticipantResponse